README.md

```diff
@@ -1,107 +1,68 @@
-
-## Intro
-
-
-tRPC allows you to easily build & consume fully typesafe APIs, without schemas or code generation.
-
-- 🧙‍♂️&nbsp; Full static typesafety & autocompletion on the client - on inputs, outputs, & errors.
-- 🐎&nbsp; Snappy DX. No code generation, run-time bloat, or build pipeline.
-- 🍃&nbsp; Light. tRPC has zero deps and a tiny client-side footprint.
-- 🐻&nbsp; Easy to add to your existing brownfield project.
-- 🔋&nbsp; Batteries included. React-library + Next.js/Express adapters. _(But tRPC is not tied to React - [reach out](https://twitter.com/alexdotjs) if you want to make a Svelte/Vue/... lib)_
-- 🥃&nbsp; Simple to use APIs for queries, mutations, & subscriptions support.
-- ⚡️&nbsp; Request batching - requests made at the same time can be automatically combined into one
-
-
-... and:
-
-- 👀&nbsp; Quite a few examples in the [./examples](./examples)-folder
-- ✅&nbsp; It's well-tested & running in production.
-
-
-> Still reading? Follow [**@alexdotjs** on Twitter](https://twitter.com/alexdotjs) if you have any questions or want to keep up to date what's coming next.
-
----
-
-- [Intro](#intro)
-- [Usage](#usage)
-- [Core Team](#core-team)
-- [Financial Contributors](#financial-contributors)
-  - [🥇 Gold Sponsors](#-gold-sponsors)
-  - [🥈 Silver Sponsors](#-silver-sponsors)
-  - [🥉 Bronze Sponsors](#-bronze-sponsors)
-  - [😻 Individuals](#-individuals)
-- [All contributors ✨](#all-contributors-)
-
-
-## Usage
-
-**👉  See full documentation documentation on [tRPC.io](https://trpc.io/docs). 👈**
-
-
-**Quick start with a full-stack Next.js example:**
-
+<p align="center">
+  <a href="https://trpc.io/"><img src="./www/static/img/logo-text-horizontal.png" alt="tRPC" height="130"/></a>
+</p>
+<p></p>
+<p align="center">
+  <strong>End-to-end typesafe APIs made easy</strong>
+</p>
+<p align="center">
+  <a href="https://trpc.io/discord">
+    <img alt="Discord" src="https://img.shields.io/discord/867764511159091230?color=7389D8&label=6A7EC2&logoColor=ffffff" />
+  </a>
+  <a href="https://codecov.io/gh/trpc/trpc">
+    <img alt="codecov" src="https://codecov.io/gh/trpc/trpc/branch/main/graph/badge.svg?token=KPPS918B0G">
+  </a>
+  <a href="https://github.com/trpc/trpc/blob/main/LICENSE">
+    <img alt="MIT License" src="https://img.shields.io/github/license/trpc/trpc" />
+  </a>
+  <a href="https://github.com/trpc/trpc">
+    <img alt="Github Stars" src="https://badgen.net/github/stars/trpc/trpc" />
+  </a>
+</p>
+
+</br>
+
+<p align="center">
+  <img src="https://storage.googleapis.com/trpc/trpcgif.gif" alt="Server/client example" />
+</p>
+<p>
+ tRPC allows you to build & consume typesafe APIs, without schemas or code generation. As TypeScript and static typing increasingly become a best practice in web programming, the API presents a major pain point. We need better ways to statically type our API endpoints and share those types between our client and server (or server-to-server).
+</p>
+
+## Features 
```

Comment:
I personally dislike the changes made here makes it a lot harder to digest. I feel that you removed quite a few important items and removing the emojis honestly just looks like a wall of text which is harder to read. The emoji’s create a clear separation between the points where the default bullets just blends together.

Reply:
I agree. Working on it...